src/SearchConfiguration.php

```diff
@@ -24,6 +28,26 @@ public function __construct(
     {
     }
 
+    /**
+     * Define wether all tokens of a search query shall use a wildcard.
+     */
+    public function usingWildcardsForAllToken(bool|string $wildcardAllTokens): self
```

Comment:
Spelling differs from configuration value:
```suggestion
    public function usingWildcardsForAllTokens(bool|string $wildcardAllTokens): self
```
Please make sure to change the other occurrences as well.